admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/data/mapper/deposit/depositMapper.kt

```diff
@@ -0,0 +1,11 @@
+package net.thechance.mena.admin_panel.data.mapper.deposit
+
+import net.thechance.mena.admin_panel.data.remote.dto.deposit.DepositRequestDto
+import net.thechance.mena.admin_panel.domain.model.DepositQueryParams
+
```

Comment:
Just a small formatting suggestion here to improve readability and match Kotlin style conventions (add spaces around = and proper indentation):

```suggestion
fun DepositQueryParams.toRequest(): DepositRequestDto {
    return DepositRequestDto(
        phoneNumber = phoneNumber,
        amount = amount
    )
}
```